openedx/core/djangoapps/user_api/accounts/serializers.py

```diff
@@ -63,6 +68,16 @@ def to_representation(self, user):
         :return: Dict serialized account
         """
         profile = user.profile
+        badges = settings.FEATURES.get('ENABLE_OPENBADGES') or False
+        if 'badges' in self.excluded_fields:
+            # Avoid doing this potentially expensive query-- it will be filtered out anyway.
```

Comment:
By the way, is there a requirement to merge this before today's release cut?  It's not a requirement for mobile.  And I believe your team's Dec deadline is for merging to master, not release to production.